src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -149,15 +149,22 @@ const ForgotPassword = ({ intl, location, dependencies: { dopplerLegacyClient }
           </p>
         </footer>
       </article>
-      <section className="feature-panel bg--forgot">
-        <article className="feature-content">
-          <h6>{_('feature_panel.forms')}</h6>
-          <h1>{_('feature_panel.forms_description')}</h1>
-          <p>{_('feature_panel.forms_remarks')}</p>
-        </article>
-        <figure className="content-img">
-          <img src={_('forgot_password.image_path')} alt="Subscription Forms" />
-        </figure>
+      <section className="feature-panel">
+        <div
+          className="feature-panel--bg"
+          style={{
+            backgroundImage: `urL('https://qa.fromdoppler.com//wp-content//uploads//2019//06//violet-yellow.png')`,
```

Comment:
Y las dobles `/` tampoco son necesaria, no?
```suggestion
            backgroundImage: `url('https://qa.fromdoppler.com/wp-content/uploads/2019//06/violet-yellow.png')`,
```

Reply:
subido